src/Umbraco.Cms.Integrations.Crm.Dynamics/App_Plugins/UmbracoCms.Integrations/Crm/Dynamics/Render/DynamicsForm.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using Umbraco.Cms.Integrations.Crm.Dynamics.Models.ViewModels;
+
+@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<FormViewModel>
+
+@{
+    var id = Guid.NewGuid();
+}
+
+@if (Model.IFrameEmbedded)
+{
+    <iframe frameBorder="0" style="width:100%" id=@id></iframe>
```

Comment:
`id` attribute should have quotes around it.

Reply:
The _iframe_ code is picked from the Dynamics private portal, adding the quotes caused the _iframe_ to fail loading.